pyccel/stdlib/lists/lists.c

```diff
@@ -219,4 +210,228 @@ void*   array_subscripting(t_list *list, size_t index)
     if (list->size > index)
         return ((void*)(&elements[index * tSizes[list->type]]));
     return (NULL);
+}
+
+// Sorting ////////////////////////////
+
```

Comment:
it would be helpful if you can some documentation for these functions, and I should do the same with my functions.

Reply:
https://github.com/pyccel/pyccel/pull/1201/commits/a885d1ba62455a7ecd5ec8e9bbb0d57400b87562